tests/test_clients.py

```diff
@@ -1959,6 +2012,50 @@ def test_update_document(self):
         new_content.seek(0)
         self.assertEqual(posted_content.read(), new_content.read())
 
+    def test_update_document_empty_content(self):
+        identification = str(uuid.uuid4())
+        properties = {
+            "creatiedatum": timezone.now(),
+            "titel": "detailed summary",
+            "auteur": "test_auteur",
+            "formaat": "txt",
+            "taal": "eng",
+            "bestandsomvang": 17,
+            "bestandsnaam": "dummy.txt",
+            "link": "https://drc.utrechtproeftuin.nl/api/v1/enkelvoudiginformatieobjecten/d06f86e0-1c3a-49cf-b5cd-01c079cf8147/download",
+            "beschrijving": "test_beschrijving",
+            "vertrouwelijkheidaanduiding": "openbaar",
+            "uuid": str(uuid.uuid4()),
+        }
+        content = io.BytesIO(b"Content before update")
+
+        document = self.cmis_client.create_document(
+            identification=identification,
+            data=properties,
+            content=content,
+            bronorganisatie="159351741",
+        )
+
+        new_content = io.BytesIO(b"")
+
+        lock = str(uuid.uuid4())
+        self.cmis_client.lock_document(drc_uuid=document.uuid, lock=lock)
+        self.cmis_client.update_document(
+            drc_uuid=document.uuid, data={}, lock=lock, content=new_content
+        )
+        updated_doc = self.cmis_client.unlock_document(
+            drc_uuid=document.uuid, lock=lock
+        )
```

Comment:
does this make sense if `bestandsomvang` remains 17 bytes? I'd expect that to be reset to 0 because the content is empty (either explicitly and then you'd need a `ValueError` somewhere, or automatically under the hood?)

Reply:
Hmm, it would probably fail validation on the Documenten API side right?

Reply:
yes, but I think multi-layered assertions/checks is wise :-)